www/templates/main-recommendations.html

```diff
@@ -0,0 +1,96 @@
+<ion-view view-title="Recommendations" class="ion-view-background">
+     <ion-nav-buttons side="left">
+      <button ng-if="screen" class="button button-icon" ng-click="refreshPage()">
+        <i class="icon ion-android-refresh"></i>
+      </button>
+     </ion-nav-buttons>
+     <style>
+      .recommendbox {
```

Comment:
This needs to move into css files instead of being inline. Again, please check the recommendations for angular1